### ISSUE TITLE
Duplication of Speed Upgrades in the Crafters

### ISSUE THREAD

**Issue by Stevesmaster101** [Edivad99/ExtraStorage#65]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When you put speed upgrades into any of the crafters and then use inventory sorter to "sort" the inventory of the crafter it will duplicate the upgrades and place them inside of the crafter. You can spam the inventory sorter sort button and just keep duplicating them creating stacks and stacks of upgrades.

Done with `inventorysorter-1.19.2-21.0.4`

### How can I reproduce this bug or crash?

1. Place any of the crafters
2. Add speed upgrades into the upgrade slots
3. Use the "sort" key from inventory sorter and it will replicate the items into the crafters inventory

### Configuration

```markdown
- Mod version:3.0.1
- Refined Storage version:1.11.1
- Forge version:43.1.57
- Minecraft version:1.19.2
```

### Anything else?

This was done in the Direwolf20 1.19 V1.0.4 Pack. It was also a bug I had found in the earlier 1.18 version but didn't report at the time. Here is a video of me recreating it as well https://youtu.be/huvPweb4o30

**Comment by Edivad99** [comment]
I know about this problem, I reported it to the creator of the inventory sorter quite a while ago. 
I saw that maybe he fixed it, now I check.

**Comment by Edivad99** [comment]
Well, a fix for 1.18.2 is on the way.
For 1.19.2 the problem is with the inventorysorter, I've already warned the creator
https://github.com/cpw/inventorysorter/issues/119#issuecomment-1373828161
